frontend/__tests__/components/cluster-settings.spec.tsx

```diff
@@ -113,20 +110,20 @@ describe('Cluster Settings page', () => {
       wrapper
         .find(HorizontalNav)
         .at(0)
-        .props().pages[0].name,
-    ).toBe('Details');
+        .props().pages[0].nameKey,
+    ).toBe('public~Details');
```

Comment:
Alternative you check this with an regex:

```suggestion
    ).toMatch('Details');
```